spec/apprentice_teacher_spec.rb

```diff
@@ -37,12 +37,12 @@
     end
 
     it "changes the phase" do
-      rey.set_phase(1)
+      rey.set_phase_teacher(1)
```

Comment:
The tests are set up to model the expected behavior we need in the code. So changing the tests is sneaky cool but not what we were looking for. How can you modify the code,  keep the original tests and get the expected behavior?